i18n/messages/nl.json

```
@@ -653,6 +653,11 @@
     "description": "Validation error for partial coordinates (lat, lng) configuration",
     "originalDefault": "You need to configure both longitude and latitude."
   },
+  "R25Vbr": {
+    "defaultMessage": "Reference lists API",
```

```suggestion
    "defaultMessage": "Referentielijsten-API",
```
here you go for the translations :)